### ISSUE TITLE
Markdown links break if the description has template string (``) in it.

### ISSUE THREAD

**Issue by Harshita-mindfire** [jonathanyeung/mark-sharp#4]
**Describe the bug**
Markdown links break if the description has template string (`) in it.

**To Reproduce**
1. Go to any .md file
2. Add the links defined below:
```md
- [This is a request](https://google.com)
- [This is a `xyz` request](https://google.com)
```

**Expected behavior**

**Screenshots**
<img width="358" alt="image" src="https://github.com/jonathanyeung/mark-sharp/assets/84971366/7651d86e-f297-4e50-8a36-e08b4a90e742">

**Platform (please complete the following information):**
 - Mac
 - Extension Version: v1.1.0

**Comment by jonathanyeung** [comment]
:wave: good find! I can repro, it looks like it's an issue with any kind of text formatters such as bold or italics - I'll try to fix by the next version.

**Comment by jonathanyeung** [comment]
Hi @Harshita-mindfire - this should be fixed in the latest version `1.2.0` which was just released.
